doc/modules/classification_threshold.rst

```diff
@@ -146,6 +146,13 @@ also possible to manually set the decision threshold using the class
 :class:`~sklearn.model_selection.FixedThresholdClassifier`. In case that you don't want
 to refit the model when calling `fit`, you can set the parameter `prefit=True`.
 
+.. note::
+  Given a calibrated classifier and a dataset is representative and large enough,
+  the optimal decision threshold is theoretically defined as in Eq. (2) of [E2001]_.
```

Comment:
This is only possible if the business score to optimize can be expressed as a weighted sum of the entries of the confusion matrix.